src/napari/utils/_env_detection.py

```diff
@@ -0,0 +1,81 @@
+import os
+from enum import auto
+from pathlib import Path
+
+from napari.utils.compat import StrEnum
+
+
+class Environment(StrEnum):
+    pip = auto()
+    uv = auto()
+    conda = auto()
+    pixi = auto()
+
+
+def get_venv_path() -> Path | None:
+    """Get the path to the current virtual environment, if any.
+
+    Returns
+    -------
+    Path | None
+        The path to the current virtual environment, or None if not in a venv.
+    """
+    venv = os.environ.get('VIRTUAL_ENV')
+    return Path(venv) if venv else None
+
+
+def get_conda_path() -> Path | None:
+    """Get the path to the current conda environment, if any.
+
+    Returns
+    -------
+    Path | None
+        The path to the current conda environment, or None if not in conda.
+    """
+    conda = os.environ.get('CONDA_PREFIX')
+    return Path(conda) if conda else None
+
+
+def chek_if_uv_venv() -> bool:
+    """Check if we are in a uv virtual environment.
```

Comment:
```suggestion
    """Check if we are in a uv environment.
```

Reply:
Let's make this a bit more generic to uv to avoid venv confusion in the function name.